app/addon_cmds/commands.py

```diff
@@ -109,9 +113,14 @@ def run_action(
 )
 @click.option(
     "--rosa",
-    help="Install/uninstall addons via ROSA cli",
-    show_default=True,
-    is_flag=True,
+    help="""
```

Comment:
Please also update the README with some examples

Reply:
Done